src/Workspaces/Core/Portable/SymbolKey/SymbolKey.AliasSymbolKey.cs

```diff
@@ -1,7 +1,8 @@
-﻿using System.Linq;
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
```

Comment:
May want to remove, if removing reliance on LINQ.

Reply:
Done.